From a4a067e5846f8ffe93bcd9540692d7f207ce8700 Mon Sep 17 00:00:00 2001
From: Christophe Marin <christophe@krop.fr>
Date: Tue, 14 Oct 2025 13:20:26 +0200
Subject: [PATCH] Fix build with Qt 6.10

The Qt 6 private targets have been split into separate CMake files since
6.9 and must be found since Qt 6.10

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ef6a4e6..f15af97 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -388,6 +388,10 @@ set(CMAKE_AUTORCC ON)
 
 find_package(Qt6 REQUIRED COMPONENTS Gui Concurrent Core Core5Compat Network PrintSupport Qml Widgets Xml LinguistTools)
 
+if(Qt6Core_VERSION VERSION_GREATER_EQUAL 6.10)
+  find_package(Qt6 REQUIRED COMPONENTS CorePrivate)
+endif()
+
 qt_standard_project_setup()
 
 if (UNIX AND NOT APPLE)
-- 
2.51.0

